docs/can-guides/commitment/recipes/credit-card-simple/credit-card-simple.md

```diff
@@ -199,12 +165,11 @@ Print out the exported values like:
 Update the __HTML__ tab to:
 
 @sourceref ./2-read-form.html
-@highlight 4-11,15,only
 
 Update the __JavaScript__ tab to:
 
 @sourceref ./2-read-form.js
-@highlight 6-10,only
+@highlight 10-17,21,only
```

Comment:
I think this one also needs to highlight the VM props that are added.

Reply:
Done!